tools/repo_parser/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM cgr.dev/chainguard/python:latest
+
+WORKDIR .
+
+COPY ./spec_finder.py ./
+VOLUME /appdir
```

Comment:
You could consider setting `ENTRYPOINT` so that the users of this image don't need to specify `spec_finder` themselves but could pass arguments directly

See also: https://docs.docker.com/reference/dockerfile/#entrypoint